app/lib/features/public_room_search/providers/public_space_info_provider.dart

```diff
@@ -1,15 +1,9 @@
-import 'package:acter/common/models/profile_data.dart';
-import 'package:acter/common/providers/sdk_provider.dart';
+import 'package:acter/common/providers/room_providers.dart';
+import 'package:acter_avatar/acter_avatar.dart';
 import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 
-final searchItemProfileData = FutureProvider.autoDispose
-    .family<ProfileData, PublicSearchResultItem>((ref, publicSpace) async {
-  final sdk = await ref.read(sdkProvider.future);
-  final size = sdk.api.newThumbSize(48, 48);
-  final avatar = await publicSpace.getAvatar(size);
-  return ProfileData(
-    publicSpace.name() ?? publicSpace.roomIdStr(),
-    avatar.data(),
-  );
+final searchItemProfileData = Provider.autoDispose
+    .family<AvatarInfo, PublicSearchResultItem>((ref, publicSpace) {
+  return ref.watch(roomAvatarInfoProvider(publicSpace.roomIdStr()));
 });
```

Comment:
ah, I don't think this is gonna work. The searchItems contain spaces and roomIds we can't look up in the roomAvatarInfoProvider :( ...